docs/building-apps/esm-views.md

```diff
@@ -0,0 +1,131 @@
+---
+parent: Building your Apps
+title: ESM Views (micro frontends)
+---
+
+modular builds packages of `"type": "view"` as
+[ES Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules),
+rewriting all of a subset of their imports to make use of a configurable
+[ESM](https://www.skypack.dev/) [CDN](https://esm.sh/). This allows users to
+implement the [microfrontend pattern](../concepts/microfrontends.md), by
+creating an artefact that can be
+[`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)ed
+at runtime by an host application, or loaded stand-alone thanks to the automatic
+generation of a index and trampoline file.
+
+## How to build
+
+Views are built with the [`modular build`](../commands/build.md) command. The
+default behaviour when building a view is:
+
+1. All the non-local `import`s in the package's `src` directory are extracted
+   and matched with their version in the package's or the package's root
+   `package.json`
+2. The main endpoint (as defined in the view's `package.json`'s `main` field)
+   and its local imports are bundled in a single file
+3. All the non-local dependencies encountered in the process are rewritten to
+   `import` from an external ESM CDN (by default https://www.skypack.dev/),
+   using the versions extracted in step 1
+4. All the local CSS is bundled in a single file
+5. The `dist` directory is generated, containing:
+   - the js bundle
+   - the css bundle
+   - a package manifest (`package.json`) file containing
+     ([see also below](#package-manifest)):
+     - the location of the js bundle (`"module"` field)
+     - the location of the css bundle (`"style"` field)
+     - an object with the whole set of dependencies and their version ranges
+       (`"dependencies"` field)
+     - an array of bundled dependencies (`bundledDependencies` field)
+   - a synthetically generated `index.html` file, linking the trampoline file
+     and the css bundle
+   - a synthetically generated trampoline file, dynamically `import`ing the js
+     bundle
+
+The view build result can either be
+[dynamically `import`ed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
+from an host application
+([including the css bundle](https://web.dev/css-module-scripts/)) or served
+statically as a standalone application (for example, using `modular serve`).
+
+## Customise the ESM CDN
```

Comment:
Either in this section or a section above, can you include a link or a blurb about why they'd need to supply a CDN template for the build at all? It might seem obvious to us as we have been talking/working on it for a while but it probably isn't clear for someone fresh to microfrontends and ESM.